Scripts/Items/Artifacts/Tools/JacobsPickaxe.cs

```diff
@@ -70,18 +70,14 @@ public JacobsPickaxe(Serial serial)
 
         public override void Serialize(GenericWriter writer)
         {
-            base.Serialize(writer);
-
+			base.Serialize(writer);
             writer.WriteEncodedInt(0); // version
         }
 
         public override void Deserialize(GenericReader reader)
         {
             base.Deserialize(reader);
-
             int version = reader.ReadEncodedInt();
-
-            _Instances.Add(this);
```

Comment:
Why is this being removed?  They won't get checked during the timer.

Reply:
Ah I thought it was just an old Deser to fix older bugged ones.